src/zenml/integrations/whylogs/secret_schemas/__init__.py

```diff
@@ -0,0 +1,28 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+"""Initialization for the Whylabs secret schema.
+
+This schema can be used to configure a ZenML secret to authenticate ZenML to
+use the Whylabs platform to automatically log all whylogs data profiles
+generated and by pipeline steps.
+"""
```

Comment:
So far I think most of our secret schemas have just been stored in the `secret/schemas` folder. I guess this makes sense, though. It's not as fundamental as an AWS schema.

Reply:
Seldon also has some secret schemas packaged in the integration, as will KServe. Would be nice to have a CLI command to list the available schemas, I'll give you that :)